docker-compose.yml

```diff
@@ -1,8 +1,12 @@
 version: "3"
 
+volumes:
+  db-data:
 services:
   db:
     image: postgres:12
+    volumes:
+      - db-data:/var/lib/postgresql/data
```

Comment:
I'm still not _entirely_ sure why we have to include this volume now when we didn't before.

What I _think_ is happening is that when we run `docker-compose run tests`, it re-creates a separate `web` container with a new data volume for the command. This new container doesn't have the migrations applied. By adding this shared volume, we "persist" the migration between commands. 

But again, even if this is the case, I'm not sure why we weren't running into this issue before.

Reply:
This makes sense indeed. I'm wondering whether we should have a `.volumes/` folder with all of them (certs chains, etc.)